pylearn2/models/tests/test_mlp.py

```diff
@@ -588,6 +635,199 @@ def test_flattener_layer():
                                    mlp_composite.get_params()[i].eval())
 
 
+def test_flattener_layer():
+    # This function tests that linear, sigmoid and softmax layers are
```

Comment:
I think you may want to turn this into a docstring formatted comment for nose printing. Most of the other tests have a short description in the docstring which helps when things don't work.